### PR TITLE
Add cuttlefish app dependency.

### PR DESCRIPTION
**Pull request by cmeiklejohn**

### CODE REVIEW

File: log/console.log [hunk 1]
```diff
@@ -0,0 +1,39 @@
+2016-10-07 21:52:34.759 [info] <0.31.0> Application lager started on node nonode@nohost
```
Comment (bieniusa) [inline log/console.log]: You probably don't want to add this file, right?

File: log/crash.log [hunk 2]
```diff
@@ -0,0 +1,40 @@
+2016-10-07 21:52:34 =ERROR REPORT====
```
Comment (bieniusa) [inline log/crash.log]: You probably also don't want to add this to the repo?

File: log/error.log [hunk 3]
```diff
@@ -0,0 +1,5 @@
+2016-10-07 21:52:34.827 [error] <0.459.0> riak_sysmon_filter:get_node_map: error badarg @ [{ets,match_object,[sys_dist,'_'],[]},{ets,tab2list,1,[{file,"ets.erl"},{line,710}]},{riak_sysmon_filter,get_node_map,0,[{file,"/Users/cmeiklejohn/Documents/antidote/_build/default/lib/riak_sysmon/src/riak_sysmon_filter.erl"},{line,378}]},{riak_sysmon_filter,init,1,[{file,"/Users/cmeiklejohn/Documents/antidote/_build/default/lib/riak_sysmon/src/riak_sysmon_filter.erl"},{line,155}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,328}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
```
Comment (bieniusa) [inline log/error.log]: Same here

Comment (cmeiklejohn) [inline log/error.log]: Fixed.

File: log/crash.log (hunk 2 above)
Comment (cmeiklejohn) [inline log/crash.log]: Fixed.

**Comment by cmeiklejohn** [comment]
Ignored log files.

**Comment by cmeiklejohn** [comment]
Shell is still failing:

```
22:39:22.511 [error] CRASH REPORT Process riak_core_ring_manager with 0 neighbours exited with reason: bad return value: {error,enoent} in gen_server:init_it/6 line 356
22:39:22.511 [error] Supervisor riak_core_sup had child riak_core_ring_manager started with riak_core_ring_manager:start_link() at undefined exit with reason bad return value: {error,enoent} in context start_error
22:39:22.512 [error] CRASH REPORT Process <0.505.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,riak_core_ring_manager,{bad_return_value,{error,enoent}}}},{riak_core_app,start,[normal,[]]}} in application_master:init/4 line 134
[os_mon] cpu supervisor port (cpu_sup): Erlang has closed
22:39:22.512 [info] Application riak_core exited with reason: {{shutdown,{failed_to_start_child,riak_core_ring_manager,{bad_return_value,{error,enoent}}}},{riak_core_app,start,[normal,[]]}}
```

**Comment by cmeiklejohn** [comment]
Dependency:

```
Christophers-MacBook-Pro:antidote cmeiklejohn$ find . -name rebar.config | xargs grep chash
./_build/default/lib/riak_core/rebar.config:  {chash, "~>0.1.1"},
```

**Comment by cmeiklejohn** [comment]
cc: @licenser we're using the riak_core_ng fork...

**Comment by bieniusa** [comment]
This is odd: In the master branch I do not get this issue with the chash library, but I do get it in the cuttlefish branch.
Both builds were done clean.

**Comment by cmeiklejohn** [comment]
One of the dependencies must be locked at a different version.  Compare the
rebar.lock files between the two branches.

Christopher

**Comment by bieniusa** [comment]
Status:
- I can build antidote and run the console via
./_build/default/rel/antidote/bin/antidote console
@aletomsic Can you confirm that this works for you on this branch?

- Travis doesn't not compile due to some C++ issues in the erocksdb dependency:
"cc1plus: error: unrecognized command line option ‘-std=c++11’cc1plus: error: unrecognized command line option ‘-std=c++11’ "

@cmeiklejohn Can we tell Travis how to interpret this? 
Where is erocksdb actually used??

**Comment by cmeiklejohn** [comment]
It doesn't look like erocksdb is currently used, so, I'd remove it as a dependency for now until it's explicitly required.

**Comment by cmeiklejohn** [comment]
@ferd Any ideas on why the shell has different behavior than console?  It appears that some of the rebar configuration options (like, making directories, etc. used when building the rel don't happen with shell, I'm curious what's the right way to handle this?)

**Comment by bieniusa** [comment]
Finally managed to get Travis working by choosing a more recent Ubuntu version which runs gcc/g++ in version 4.8.6.

One of the Travis tests is still failing, but the issue known to us: the log-recovery test...

**Comment by cmeiklejohn** [comment]
:+1:, nice work.

**Comment by aletomsic** [comment]
it works for me too.
thanks for taking care of it.

**Pull request merged**